src/App.jsx

```diff
@@ -45,14 +40,14 @@ export default function App() {
   return (
     <>
       <header>Theme Creator</header>
-      <Form onSubmit={handleAddTheme} isEdit={false} />
+      <Form onThemeSubmit={handleAddTheme} isEdit={false} />
```

Comment:
with the `on...` props we try to mimic the naming convention of the built in react elements like button or form, therefore the prop name `onSubmit` would be ideal.